src/unix/linux_like/linux/mod.rs

```diff
@@ -1583,7 +1583,7 @@ s_no_extra_traits! {
         pub can_dlc: u8,
```

Comment:
While you're here, mind adding a comment ``// FIXME(1.0): this field was renamed to `can_len` `` based on the docs?

Reply:
Sure!

Reply:
Added!